### PR TITLE
LPS-202145 Unable to edit the Maximum Number of Characters for a Long Text field to a number greater than 280

### PR DESCRIPTION
**Pull request by Artartur**
Related Issue: [LPS-202145](https://liferay.atlassian.net/browse/LPS-202145)

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @rebsilva ,
I'm assigning this PR to you because it is frontend. Feel comfortable assigning to anybody from your squad :+1:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/MaxLengthProperties.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ export function MaxLengthProperties({
 							onSettingsChange({
 								name: 'maxLength',
 								value:
-									Number(value) <= 280
+									Number(value) <= defaultMaxLength
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/MaxLengthProperties.tsx:134]: Good job with this PR @Artartur! The changes made achieved the expected result:


https://github.com/liferay-objects/liferay-portal/assets/61854510/11b68f3a-696a-4a90-9389-835bed551cd2

**Review by igor-franca — APPROVED** [review]

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by rebsilva** [comment]
Thanks @igor-franca for the review! :tada:

**Comment by igor-franca** [comment]
ci:forward:force

**Pull request closed without merging**